src/routing.js

```diff
@@ -0,0 +1,76 @@
+import {chain, merge, toPairs, trim} from 'lodash';
+
+const isParam = segment => segment.substr(0, 1) === ':';
+const toSegments = pathname => trim(pathname, '/').split('/');
+
+const buildRoute = (segments, methods) => {
```

Comment:
Can you add some documentation to this method, e.g.

- Short explanation what this method does
- What types should the input parameters have
- What's the return type